packages/augur-ui/src/modules/alerts/components/alerts-view.tsx

```diff
@@ -1,107 +1,97 @@
-import React, { Component } from "react";
+import React from "react";
 import classNames from "classnames";
 
 import NullStateMessage from "modules/common/null-state-message";
 import Alert from "modules/alerts/components/alert";
-
+import {Alert as AlertType} from 'modules/types'
```

Comment:
Can you just import as Alert

Reply:
We already used  the alert Keyword in Line 5 for a component import.